database/database.go

```diff
@@ -51,15 +51,20 @@ func New(fluidkeysDirectory string) Database {
 // RecordFingerprintImportedIntoGnuPG takes a given fingperprint and records that it's been
 // imported into GnuPG by writing an updated json database.
 func (db *Database) RecordFingerprintImportedIntoGnuPG(newFingerprint fpr.Fingerprint) error {
```

Comment:
I'm struggling to work out what's happening in this commit: there seems to be quite a lot of changes, and the commit message doesn't give me many clues!